pkgs/profpatsch/nman/nman.rs

```diff
@@ -7,11 +7,11 @@ use std::process::Command;
 use temp::TempDir;
 
 enum NmanError<'a> {
-    NoTempDir,
+    IO,
```

Comment:
you can also pass the IO error message as string or structured type

Reply:
Done in ccf42310a3a83b32277455ab0e4d36c6e9ecbe70